lib/shopify_api/session.rb

```diff
@@ -91,8 +91,10 @@ def request_token(params)
 
       if response.code == "200"
         token = JSON.parse(response.body)['access_token']
+      elsif response.code == "400"
+        raise ShopifyAPI::ValidationException, "Invalid Code: Possible malicious login"
       else
-        raise RuntimeError, response.msg
+        raise ShopifyAPI::ValidationException, "Invalid Request: Unable to validate code"
```

Comment:
I am okay with this change but can we rename the exception to `ShopifyAPI::SessionException` validation exception makes me think of other things.

Also can we keep the response message for the else case